ska_conda/pkg_defs/ska3-template/bin/flt_envs

```diff
@@ -28,28 +27,18 @@ if SHELL not in ['csh', 'tcsh', 'bash', 'sh', 'ksh']:
     raise ValueError("shell '%s; not supported" % SHELL)
 
 
-uname_tuple = platform.uname()
-# platform.uname returns a tuple of strings (system,node,release,version,machine,processor)
-# uname is intended to be like linux-x86_64
-system, machine = uname_tuple[0].lower(), uname_tuple[4].lower()
-system = {'darwin': 'osx'}.get(system, system)
-machine = {'x86_64': '64', 'i686': '32'}.get(machine, machine)
-uname = '%s-%s' % (system, machine)
-
 # Use the convention that flt_envs must be in $SKA/bin to derive SKA and
 # OVERRIDE any existing value.
 ska = dirname(dirname(abspath(__file__)))  # from $SKA/bin/flt_envs to $SKA
 
 # Set environment vars
 envs = dict(('SKA_' + suffix.upper(), os.path.join(ska, suffix))
         for suffix in ('arch', 'data', 'lib', 'share', 'bin', 'idl'))
-ska_arch_os = os.path.join(ska, 'arch', uname)
-envs['SKA_ARCH_OS'] = ska_arch_os
```

Comment:
For the No arch install, do we have any reason to define SKA_ARCH_OS = SKA or is it best to get rid of it altogether as I've done here?